examples/wCCD/src/Root.tsx

```diff
@@ -17,10 +17,26 @@ export default function Root() {
         setIsConnected(Boolean(accountAddress));
     }, []);
 
+    const handleNotConnected = useCallback(() => {
+        setAccount('');
+        setIsConnected(false);
+    }, []);
+
     useEffect(() => {
         detectConcordiumProvider()
             .then((provider) => {
                 // Listen for relevant events from the wallet.
+                provider.on('chainChanged', (genesisBlock) => {
+                    // Check if the user is connected to testnet by checking if the genesisBlock is the testnet one.
+                    // Throw a warning and disconnect if wrong chain. We only want to
+                    // allow users to interact with our testnet smart contracts.
+                    if (genesisBlock !== '4221332d34e1694168c2a0c0b3fd0f273809612cb13d000d5c2e00e85f50f796') {
```

Comment:
Is it possible to put this in a named, documented constant so that it does not appear misteriously in two places?